Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -153,7 +153,15 @@ class BrowserViewController: UIViewController {
         rewardsObserver = nil
         #else
         RewardsHelper.configureRewardsLogs()
-        rewards = BraveRewards(configuration: .default)
+        if AppConstants.BuildChannel.isRelease {
+            rewards = BraveRewards(configuration: .production)
+        } else {
+            if let override = Preferences.Rewards.EnvironmentOverride(rawValue: Preferences.Rewards.environmentOverride.value), override != .none {
+                rewards = BraveRewards(configuration: override == .prod ? .production : .default)
```

Comment:
Does this mean we can use the production BraveRewards in the debug builds?

Reply:
Yes, you can override the environment to prod and after you relaunch the app you can create a wallet on prod environment and it'll download the prod pub list